packages/components/src/EditableText/EditableText.component.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { translate } from 'react-i18next';
+import Skeleton from '../Skeleton';
+import { Action } from '../Actions';
+import InlineForm from './InlineForm.component';
+import theme from './EditableText.scss';
+import I18N_DOMAIN_COMPONENTS from '../constants';
+
+function EditableText({ text, editMode, loading, inProgress, onEdit, disabled, t, ...rest }) {
```

Comment:
EditableText should be used in SubHeaderBar

Reply:
✅